<?xml version="1.0" encoding="utf-8"?>
<launch>
	<node pkg="ros_cv" type="ros_cv_node" name="camera" output="screen" respawn="true">	
	<!-- 测试视频 -->
	<!-- <param name="videoname" value= "/home/xx/andyoyo/usv_ImagePro/src/ros_cv/data/lib2.avi"/> -->
	<!-- 网络相机 -->
	<param name="videoname" value= "rtsp://admin:txz123456@192.168.1.66:554/mjpeg/ch1/sub/av_stream"/>
	</node>	
	<node pkg="yolocv_kcf" type="yolocv_kcf_node" name="yolocv_kcf" clear_params="true" output="screen" respawn="true">	
	<!-- First way:direct set (自己训练的模型文件)-->
	<!-- <param name="modelConfig" value= "$(find yolocv_kcf)/data/mycfg/yolov3-tiny_xx.cfg"/> -->
	<!-- <param name="modelWeights" value= "$(find yolocv_kcf)/data/mycfg/yolov3-tiny_xx_100000.weights"/> -->
	<!-- <param name="modelNames" value= "$(find yolocv_kcf)/data/mycfg/voc_xx.names"/> -->
	<!-- Second way:use .yaml file (yolo官方的模型文件)-->
	<rosparam file="$(find yolocv_kcf)/yaml/yolo3cfg.yaml"/>
	</node>	
</launch>